tests/unit/test_property_based.py

```diff
@@ -0,0 +1,79 @@
+import string
+
+import hypothesis as hp
+import pandas as pd
+from hypothesis import strategies as st
+from hypothesis.extra.pandas import data_frames, columns, range_indexes
+
+from shmessy import Shmessy
+
+max_examples = 20
+
+
+@st.composite
+def df_st(draw) -> st.SearchStrategy[pd.DataFrame]:
+    col_names = draw(st.sets(st.text(min_size=1, max_size=10), min_size=2, max_size=5))
+    dfs_st = data_frames(
+        columns=columns(
+            list(col_names),
+            dtype=draw(st.sampled_from([
+                # int,    # todo fix shmessy f or int type. fails by exception
```

Comment:
Int / float types should work as expected, please check the latest release (1.1.8)